--- rosegarden-4-0.9.1/base/Studio.C.orig	Sat May 31 13:13:07 2003
+++ rosegarden-4-0.9.1/base/Studio.C	Tue Oct 21 10:21:29 2003
@@ -568,7 +568,7 @@
 bool
 Studio::haveMidiDevices() const
 {
-    Rosegarden::DeviceListIterator it = m_devices.begin();
+    DeviceListConstIterator it = m_devices.begin();
     for (; it != m_devices.end(); it++)
     {
         if ((*it)->getType() == Device::Midi) return true;
