--- rosegarden-4-0.9/gui/rosegarden.desktop.fix	Fri May  2 18:25:24 2003
+++ rosegarden-4-0.9/gui/rosegarden.desktop	Fri May  2 18:27:51 2003
@@ -10,3 +10,4 @@
 Terminal=0
 Name=Rosegarden
 Name[de]=Rosegarden
+Categories=Application;AudioVideo;
\ No newline at end of file
--- rosegarden-4-0.9/gui/Makefile.in.fix	Fri May  2 18:25:44 2003
+++ rosegarden-4-0.9/gui/Makefile.in	Fri May  2 18:26:45 2003
@@ -1277,10 +1277,10 @@
 install-data-local:
 #>- 	$(mkinstalldirs) $(kde_appsdir)/Applications/
 #>+ 1
-	$(mkinstalldirs) $(DESTDIR)$(kde_appsdir)/Applications/
+	$(mkinstalldirs) $(DESTDIR)/usr/share/applications/
 #>- 	$(INSTALL_DATA) $(srcdir)/rosegarden.desktop $(kde_appsdir)/Applications/rosegarden.desktop
 #>+ 1
-	$(INSTALL_DATA) $(srcdir)/rosegarden.desktop $(DESTDIR)$(kde_appsdir)/Applications/rosegarden.desktop
+	$(INSTALL_DATA) $(srcdir)/rosegarden.desktop $(DESTDIR)/usr/share/applications/rosegarden.desktop
 #>- 	$(mkinstalldirs) $(kde_icondir)/locolor/32x32/apps/
 #>+ 1
 	$(mkinstalldirs) $(DESTDIR)$(kde_icondir)/locolor/32x32/apps/
