--- rosegarden-1.6.1/data/desktop/rosegarden.desktop.orig	2008-03-31 03:52:27.489681034 +0200
+++ rosegarden-1.6.1/data/desktop/rosegarden.desktop	2008-03-31 03:55:50.975695904 +0200
@@ -1,13 +1,14 @@
 [Desktop Entry]
 Encoding=UTF-8
 Type=Application
-Categories=AudioVideo;Audio;Midi;Sequencer;X-SuSE-Sequencer;X-Red-Hat-Base;
+Categories=Qt;KDE;AudioVideo;Audio;Midi;Sequencer;
 Exec=rosegarden -caption "%c" %i %m
 MimeType=audio/x-rosegarden;audio/x-rosegarden-device
 X-KDE-NativeMimeType=audio/x-rosegarden
 Icon=rosegarden.png
 DocPath=rosegarden/index.html
 Comment=MIDI and Audio Sequencer and Editor
+Comment[pl]=Sekwencer i edytor MIDI i dźwięku
 Terminal=false
 Name=Rosegarden
 Name[de]=Rosegarden
