diff -aurN rosegarden-4-1.2.3.orig/SConstruct rosegarden-4-1.2.3/SConstruct
--- rosegarden-4-1.2.3.orig/SConstruct	2006-02-14 08:55:21.000000000 +0100
+++ rosegarden-4-1.2.3/SConstruct	2006-05-22 22:56:15.278199000 +0200
@@ -142,22 +142,22 @@
 	env.KDEinstall('KDEMIME','/audio', mimefiles)
 
 	# icons
-	env.KDEinstallas('KDEICONS','/locolor/16x16/apps/x-rosegarden.xpm', "gui/pixmaps/icons/cc-hi16-rosegarden.xpm")
-	env.KDEinstallas('KDEICONS','/hicolor/16x16/apps/x-rosegarden.xpm', "gui/pixmaps/icons/rg-rwb-rose3-16x16.png")
+	env.KDEinstall('KDEICONS','/locolor/16x16/apps/x-rosegarden.xpm', "gui/pixmaps/icons/cc-hi16-rosegarden.xpm")
+	env.KDEinstall('KDEICONS','/hicolor/16x16/apps/x-rosegarden.xpm', "gui/pixmaps/icons/rg-rwb-rose3-16x16.png")
 
-	env.KDEinstallas('KDEICONS','/hicolor/48x48/apps/rosegarden.png', "gui/pixmaps/icons/rg-rwb-rose3-48x48.png")
-	env.KDEinstallas('KDEICONS','/hicolor/64x64/apps/rosegarden.png', "gui/pixmaps/icons/rg-rwb-rose3-64x64.png")
+	env.KDEinstall('KDEICONS','/hicolor/48x48/apps/rosegarden.png', "gui/pixmaps/icons/rg-rwb-rose3-48x48.png")
+	env.KDEinstall('KDEICONS','/hicolor/64x64/apps/rosegarden.png', "gui/pixmaps/icons/rg-rwb-rose3-64x64.png")
 
-	env.KDEinstallas('KDEICONS','/hicolor/128x128/apps/rosegarden.png', "gui/pixmaps/icons/rg-rwb-rose3-128x128.png")
+	env.KDEinstall('KDEICONS','/hicolor/128x128/apps/rosegarden.png', "gui/pixmaps/icons/rg-rwb-rose3-128x128.png")
 
-	env.KDEinstallas('KDEICONS','/locolor/32x32/apps/rosegarden.xpm', "gui/pixmaps/icons/cc-hi32-rosegarden.xpm")
-	env.KDEinstallas('KDEICONS','/hicolor/32x32/apps/rosegarden.png', "gui/pixmaps/icons/rg-rwb-rose3-32x32.png")
+	env.KDEinstall('KDEICONS','/locolor/32x32/apps/rosegarden.xpm', "gui/pixmaps/icons/cc-hi32-rosegarden.xpm")
+	env.KDEinstall('KDEICONS','/hicolor/32x32/apps/rosegarden.png', "gui/pixmaps/icons/rg-rwb-rose3-32x32.png")
 
-	env.KDEinstallas('KDEICONS','/hicolor/16x16/mimetypes/x-rosegarden.png', "gui/pixmaps/icons/mm-mime-hi16-rosegarden.png")
-	env.KDEinstallas('KDEICONS','/locolor/16x16/mimetypes/x-rosegarden.png', "gui/pixmaps/icons/mm-mime-hi16-rosegarden.png")
+	env.KDEinstall('KDEICONS','/hicolor/16x16/mimetypes/x-rosegarden.png', "gui/pixmaps/icons/mm-mime-hi16-rosegarden.png")
+	env.KDEinstall('KDEICONS','/locolor/16x16/mimetypes/x-rosegarden.png', "gui/pixmaps/icons/mm-mime-hi16-rosegarden.png")
 
-	env.KDEinstallas('KDEICONS','/hicolor/32x32/mimetypes/x-rosegarden.png', "gui/pixmaps/icons/mm-mime-hi32-rosegarden.png")
-	env.KDEinstallas('KDEICONS','/locolor/32x32/mimetypes/x-rosegarden.png', "gui/pixmaps/icons/mm-mime-hi32-rosegarden.png")
+	env.KDEinstall('KDEICONS','/hicolor/32x32/mimetypes/x-rosegarden.png', "gui/pixmaps/icons/mm-mime-hi32-rosegarden.png")
+	env.KDEinstall('KDEICONS','/locolor/32x32/mimetypes/x-rosegarden.png', "gui/pixmaps/icons/mm-mime-hi32-rosegarden.png")
 
 	# styles
 	#for s in glob.glob("gui/styles/*.xml"):
